docker/docker-compose-centos-dev.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, EMC, Inc.
```

Comment:
This file comes from a customer request, they want to run RackHD inside docker over CentOS.